### PR TITLE
BUGFIX: Adjust functional RoutingTest

### PR DESCRIPTION
**Pull request by kdambekalns**

### CODE REVIEW

**Review by bwaidelich — APPROVED** [review]
Thanks!

File: Neos.Flow/Configuration/Testing/Settings.yaml [hunk 1]
```diff
@@ -49,8 +49,9 @@ Neos:
                 logFileURL: '%FLOW_PATH_DATA%Logs/I18n_Testing.log'
                 severityThreshold: '%LOG_DEBUG%'
 
-
-
+    mvc:
+      routes:
+        'Neos.Flow': true
```
Comment (bwaidelich) [inline Neos.Flow/Configuration/Testing/Settings.yaml:54]: This will break functional tests in the Neos distribution

**Review by bwaidelich — CHANGES_REQUESTED** [review]
It would be good to include Flow routes by default in testing context, but we probably have to do that in the CI for now

File: Neos.Flow/Configuration/Testing/Settings.yaml (hunk 1 above)
Comment (kdambekalns) [reply Neos.Flow/Configuration/Testing/Settings.yaml]: Dropped the commit again…

**Review by bwaidelich — APPROVED** [review]

**Review by albe — APPROVED** [review]

**Pull request merged**